.m2-tmp/com/liferay/com.liferay.project.templates/4.1.7/com.liferay.project.templates-4.1.7.pom

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Hey @Camork  there is no need to send a FAKE GRADLE CACHE for the project-templates.  Only need to send one for the project template itself.